### ISSUE TITLE
regd. touch2success.com image URL

### ISSUE THREAD

**Issue by karthik-t2s** [Tagpoetry/Hotwings#1]
Hi
I writing to you behalf of Touch2success.com, and i would like to let you know that your repository ("Hotwings") contains our public image URL **(https://public.touch2success.com/static/30c4ef1f588326a8107818c508e4f755/img/phpxHICjb.png)**

Repo URLs
https://github.com/Tagpoetry/Hotwings/blob/156cbfdd6076933a0d021088d741e53096338c3a/Store/top6stlwingspots.js
https://github.com/Tagpoetry/Hotwings/blob/e2113fbdd567211efd4b983ded15141789350a50/top10stlwingspots.html
https://github.com/Tagpoetry/Hotwings/blob/911fd631bd353c9b9e2d3330110087194770b46e/Iheard.html

Could you please remove those image URLs.

Appreciate your help.
Thanks
Karthik J

**Comment by Tagpoetry** [comment]
okay, i have to figure out how to get into my repository again and take it
down. i had made that website for a school project.

**Comment by bhuvan-t2s** [comment]
Hello,

Did you get a chance to check and remove the files that were mentioned already

**Comment by Tagpoetry** [comment]
Yes, I took down the entire website. Thank you again for reaching out to me.

**Comment by bhuvan-t2s** [comment]
Hello,

Good Morning!

Thanks for the update. Still Tagpoetry/Hotwings repository has the URL's that was mentioned already. Could you please check and remove repo "Tagpoetry/Hotwings".

Please consider it as urgent.

**Comment by bhuvan-t2s** [comment]
Hello,

Good Morning!

Thanks for the update. Still Tagpoetry/Hotwings repository has the URL's that was mentioned already. Could you please check and remove repo named "Tagpoetry/Hotwings".

Please consider it as urgent.
